### PR TITLE
Make travis not time out

### LINKED ISSUE #27: Travis-CI build fails on linting
The Travis-CI build fails on the after_script linting. Because there are so many lint errors that Travis eventually times out.

Possible Travis bug: Should timing out in an after_script action fail a build?

### PR DESCRIPTION
**Pull request by diracdeltas**
Fix #27

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

**Comment by diracdeltas** [comment]
Yay, travis is happy. @coruus please review/merge.

File: do.sh [hunk 1]
```diff
@@ -228,7 +228,7 @@ e2e_lint() {
     else
       ADDITIONAL=$*
     fi
-    gjslint --strict --closurized_namespaces=goog,e2e --limited_doc_files=_test.js $ADDITIONAL
+    gjslint --strict --closurized_namespaces=goog,e2e --limited_doc_files=_test.js --exclude_files=src/javascript/crypto/e2e/extension/utils/wordlist.js $ADDITIONAL
```
Comment (coruus) [inline do.sh:231]: Ahhh, indeed.

File: src/javascript/crypto/e2e/extension/utils/passphrase_test.js [hunk 2]
```diff
@@ -49,7 +49,8 @@ function testBytesToPhrase() {
   var evenBytes =
       [0, 0, 17, 212, 12, 140, 90, 247, 46, 83, 254, 60, 54, 169, 255, 255];
   var phrase =
-      'a bioengineering balloted gobbledegook creneled written depriving zyzzyva';
+      'a bioengineering balloted gobbledegook creneled written depriving ' +
+      'zyzzyva';
```
Comment (coruus) [inline src/javascript/crypto/e2e/extension/utils/passphrase_test.js:53]: Wait, zyzzyva, seriously?

**Pull request merged**